src/utils/data.ts

```diff
@@ -0,0 +1,58 @@
+import { setHours, setMinutes } from 'date-fns';
+import { Doctor, UserVisit } from './types';
+
+export const doctors: Doctor[] = [
```

Comment:
to by musiało do db_json lecieć

Reply:
cos wiecej?

Reply:
Wydaje mi się że tę strukturę danych zmieniasz podczas działania aplikacji, więc żeby to wyglądało profesjonalnie i żeby stan aplikacji był gdzieś trzymany po zamknięciu przeglądarki musiałoby to lecieć do bazy

Reply:
tak tak ale jak nie robie projektu z supabase to to odpuscilem juz (moglem do localstorage itd)

Ps
tak zaluje, ze nie wrzucilem tego do supabase :D